src/coreclr/jit/barrierkinddef.h

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
perhaps easier to just move this to compiler.h ?

Reply:
I can if you want - we have `#include "compiler.h" // temporary??` at the top of `codegen.h` though so I assumed we wanted it somewhere that's able to be included separately. Also, where should I put it in `compiler.h`? Just at the bottom?

Reply:
It looks like `// temporary??` has been there since the initial repo population (year 2015) and may have been present for quite some time before that :-) 
The context for this comment is probably lost by now (is it suggesting something or just noticing?)  

> Also, where should I put it in compiler.h? Just at the bottom?

I think at the bottom would be fine.

Reply:
I had to put it further up, since we use it in `compiler.h` - let me know if it's a bad spot and I'll move it.